.github/workflows/test.yml

```diff
@@ -346,9 +347,9 @@ jobs:
       # Get the latest uploaded txt with the disk SHORT_SHA from this workflow
       - name: Download latest disk state SHORT_SHA
         uses: dawidd6/action-download-artifact@v2.17.0
-        # Just search for the latest uploaded artifact if the previous job was skipped,
+        # Just search for the latest uploaded artifact if the previous disk regeneration job was skipped,
         # otherwise use the output from ${{ needs.regenerate-stateful-disks.outputs.disk_short_sha }}
-        if: ${{ needs.regenerate-stateful-disks.steps.create-instance.outcome == 'skipped' }}
+        if: ${{ needs.regenerate-stateful-disks.outputs.any_changed != 'true' ||  || github.event.inputs.regenerate-disks != 'true'}}
```

Comment:
Do you need two sets of `||` here?
<img width="889" alt="image" src="https://user-images.githubusercontent.com/552961/156715760-5c238ddf-aeab-4c39-94b6-e81fd61bc440.png">

Reply:
Oops! [bcfea78](https://github.com/ZcashFoundation/zebra/pull/3735/commits/bcfea783662fff10c3d2c81b2013749195c4eebd)